blueprints.yaml

```diff
@@ -1,5 +1,5 @@
 name: Google Analytics
-version: 1.4.0
+version: 2.0~b1
```

Comment:
Why such version name? I'd like to follow semantic versioning. What does `~b1` mean?

Reply:
> Can you please confirm this is backwards compatible?

No, I cannot test it: I have no access to older Google Analytics accounts, so I am unable to test it.